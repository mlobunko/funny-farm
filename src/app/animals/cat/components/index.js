import Cat from './CatContainer';

export default Cat;
