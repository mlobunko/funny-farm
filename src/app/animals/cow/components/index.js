import Cow from './CowContainer';

export default Cow;
