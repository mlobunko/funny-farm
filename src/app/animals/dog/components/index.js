import Dog from './DogContainer';

export default Dog;
