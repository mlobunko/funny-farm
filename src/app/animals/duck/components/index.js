import Duck from './DuckContainer';

export default Duck;
