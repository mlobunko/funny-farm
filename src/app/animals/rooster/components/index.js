import Rooster from './RoosterContainer';

export default Rooster;
