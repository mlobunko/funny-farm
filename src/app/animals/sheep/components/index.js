import Sheep from './SheepContainer';

export default Sheep;
