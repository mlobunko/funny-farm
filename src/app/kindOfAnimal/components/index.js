import KindOfAnimal from './KindOfAnimalContainer';

export default KindOfAnimal;
