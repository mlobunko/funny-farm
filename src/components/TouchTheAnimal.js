import React from "react";

const TouchTheAnimal = () => (
  <div className="touch-the-animal">Touch The Animal</div>
);

export default TouchTheAnimal;
