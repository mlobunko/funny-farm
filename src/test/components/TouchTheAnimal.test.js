import React from "react";
import { shallow } from "enzyme";
import { TouchTheAnimal } from "../../components/TouchTheAnimal";

it("should render TouchTheAnimal component correctly", () => {
  const wrapper = shallow(<TouchTheAnimal />);
  expect(wrapper).toMatchSnapshot();
});
