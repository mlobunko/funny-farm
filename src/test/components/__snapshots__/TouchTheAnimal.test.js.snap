// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TouchTheAnimal component correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="touch-the-animal touch-the-animal--animation"
>
        Touch The Animal
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Touch The Animal",
        "className": "touch-the-animal touch-the-animal--animation",
      },
      "ref": null,
      "rendered": "Touch The Animal",
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="touch-the-animal touch-the-animal--animation"
>
          Touch The Animal
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Touch The Animal",
          "className": "touch-the-animal touch-the-animal--animation",
        },
        "ref": null,
        "rendered": "Touch The Animal",
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
